hoot-rnd/src/main/cpp/hoot/rnd/perty/PertyDuplicatePoiOp.h

```diff
@@ -83,11 +83,6 @@ class PertyDuplicatePoiOp : public OsmMapOperation, public Configurable, public
 
   virtual void setRng(boost::minstd_rand& rng) { _rng = &rng; }
 
-  /**
-    @see OsmMapOperation
-    */
-  QString toString();
-
   virtual QString getDescription() const
```

Comment:
I replaced the toString function on OsmMapOperation with the two new status message functions.  Perty classes were the only ones making use of toString were using it in kind of an unuseful and inconsistent way.